clients/java/src/main/java/io/camunda/zeebe/client/impl/command/CreateUserCommandImpl.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.client.impl.command;
+
+import io.camunda.zeebe.client.api.JsonMapper;
+import io.camunda.zeebe.client.api.ZeebeFuture;
+import io.camunda.zeebe.client.api.command.CreateUserCommandStep1;
+import io.camunda.zeebe.client.api.command.FinalCommandStep;
+import io.camunda.zeebe.client.api.response.CreateUserResponse;
+import io.camunda.zeebe.client.impl.http.HttpClient;
+import io.camunda.zeebe.client.impl.http.HttpZeebeFuture;
+import io.camunda.zeebe.client.protocol.rest.UserWithPasswordRequest;
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+import org.apache.hc.client5.http.config.RequestConfig;
+
+public final class CreateUserCommandImpl implements CreateUserCommandStep1 {
+
+  private final UserWithPasswordRequest request;
+  private final JsonMapper jsonMapper;
+  private final HttpClient httpClient;
+  private final RequestConfig.Builder httpRequestConfig;
+
+  public CreateUserCommandImpl(final HttpClient httpClient, final JsonMapper jsonMapper) {
+    this.jsonMapper = jsonMapper;
+    this.httpClient = httpClient;
+    httpRequestConfig = httpClient.newRequestConfig();
+    request = new UserWithPasswordRequest();
```

Comment:
💭 I'm not thinkin about the `UserWithPasswordRequest` name. Will there be a user request without password in it? I feel like a better name would be `UserCreationReqeust` as that seems more in line with the naming we have for other requests. WDYT?

Reply:
Naming, the most famous problem with devs :D I agree to change to be more consistent but it's not only UserCreationRequest, we were using the same structure for updating a user.

Reply:
That's fair. We can call it a UserRequest?